kafka2/README.md

```diff
@@ -64,7 +64,7 @@ python getmessages_kafka2.py -t
 
 ###### If satisfied with the output, configure the agent to run continuously:
 ```bash
-sudo ./setup/cron-config.sh
```

Comment:
@zhangzinan Why this change?

Reply:
1. This agent will exit if no msg received after a certain time. So need monit to start it again.
2. This agent need only start one process to consume one topic

Reply:
We need to check the cron config and make sure it does the following:
- Check every 60 seconds to see if the process is running
-- If the process is not running, start it.
-- else, do nothing

Reply:
when I install the cron, and it seems does not doing what we want: 
![Screenshot from 2020-04-22 11-23-52](https://user-images.githubusercontent.com/41305155/80003104-86340500-848e-11ea-83d8-ea5fddc9a3b9.png)